frontend/src/metabase/visualizations/components/ChartClickActions.css

```diff
@@ -0,0 +1,34 @@
+.token {
```

Comment:
In general I think we're moving away from the CSS modules pattern (this is the first new one I've seen in a while). I think I'd prefer that we make these tokens globally available since it seems like they'd have potential beyond just this context.